### PR TITLE
Contrail311 fixes

### PR DESCRIPTION
**Pull request by pjediny**
- WebUI cassandra port parameterization, so we can use 9042 for new installation and 9160 for old ones.
- According to https://github.com/Juniper/contrail-controller/wiki/RBAC we should use aaa_mode = no-auth so we can use WebUI

### CODE REVIEW

File: metadata/service/control/cluster.yml [hunk 1]
```diff
@@ -143,6 +144,8 @@ parameters:
         id: 2
       - host: ${_param:cluster_node03_address}
         id: 3
+      database:
+        cassandra_port: ${_param:opencontrail_webui_cassandra_port}
```
Comment (cznewt) [inline metadata/service/control/cluster.yml]: split this to engine, host, port

File: opencontrail/files/3.0/config.global.js [hunk 2]
```diff
@@ -187,7 +187,7 @@ config.redis_password = '';
 /* Cassandra Server */
 config.cassandra = {};
 config.cassandra.server_ips = [{%- for member in web.members %}'{{ member.host }}'{% if not loop.last %},{% endif %}{%- endfor %}];
-config.cassandra.server_port = '9160';
+config.cassandra.server_port = '{{ web.database.cassandra_port }}';
```
Comment (cznewt) [inline opencontrail/files/3.0/config.global.js]: and web.database.port here

File: opencontrail/files/3.0/contrail-analytics-api.conf [hunk 3]
```diff
@@ -11,6 +11,8 @@ log_level = SYS_NOTICE
 log_category = 
 log_file = /var/log/contrail/contrail-analytics-api.log
 
+aaa_mode = {{ collector.api.get('aaa_mode', 'no-auth') }}
```
Comment (cznewt) [inline opencontrail/files/3.0/contrail-analytics-api.conf]: dependent on version?

Comment (pjediny) [reply opencontrail/files/3.0/contrail-analytics-api.conf]: This is for @pupapaik to decide, he has access to contrail 3.0.x to test it. This renders only for version 3, so it should not break version 2.

**Comment by pjediny** [comment]
Amended changes proposed by @cznewt

File: opencontrail/files/3.0/contrail-analytics-api.conf (hunk 3 above)
Comment (pupapaik) [reply opencontrail/files/3.0/contrail-analytics-api.conf]: remove api section - collector.get('aaa_mode', 'no-auth')

File: metadata/service/control/analytics.yml [hunk 4]
```diff
@@ -21,6 +22,8 @@ parameters:
     collector:
       version: ${_param:opencontrail_version}
       enabled: true
+      api:
```
Comment (pupapaik) [inline metadata/service/control/analytics.yml]: remove section

File: metadata/service/control/analytics.yml [hunk 5]
```diff
@@ -21,6 +22,8 @@ parameters:
...
+        aaa_mode: ${_param:opencontrail_api_aaa_mode}
```
Comment (pupapaik) [inline metadata/service/control/analytics.yml]: put value aaa_mode: no-auth

**Review by pupapaik — CHANGES_REQUESTED** [review]

File: metadata/service/control/cluster.yml [hunk 6]
```diff
@@ -5,6 +5,7 @@ classes:
 parameters:
   _param:
     opencontrail_version: 2.2
+    opencontrail_web_database_port: 9160
```
Comment (pupapaik) [inline metadata/service/control/cluster.yml]: remove

File: metadata/service/control/cluster.yml [hunk 7]
```diff
@@ -143,6 +145,9 @@ parameters:
         id: 2
       - host: ${_param:cluster_node03_address}
         id: 3
+      database:
+        engine: cassandra
+        port: ${_param:opencontrail_web_database_port}
```
Comment (pupapaik) [inline metadata/service/control/cluster.yml]: 9160

File: metadata/service/control/control.yml [hunk 8]
```diff
@@ -124,6 +125,9 @@ parameters:
         id: 2
       - host: ${_param:cluster_node03_address}
         id: 3
+      database:
+        engine: cassandra
+        port: ${_param:opencontrail_web_database_port}
```
Comment (pupapaik) [inline metadata/service/control/control.yml]: 9160

File: metadata/service/control/single.yml [hunk 9]
```diff
@@ -5,6 +5,7 @@ classes:
 parameters:
   _param:
     opencontrail_version: 2.2
+    opencontrail_web_database_port: 9160
```
Comment (pupapaik) [inline metadata/service/control/single.yml]: remove

File: metadata/service/control/single.yml [hunk 10]
```diff
@@ -122,6 +124,9 @@ parameters:
       members:
       - host: ${_param:single_address}
         id: 1
+      database:
+        engine: cassandra
+        port: ${_param:opencontrail_web_database_port}
```
Comment (pupapaik) [inline metadata/service/control/single.yml]: 9160

**Review by pupapaik — CHANGES_REQUESTED** [review]

**Pull request merged**